blocked-edges/4.15.0-OVNIPsecPausedMCPConnectivity.yaml

```diff
@@ -0,0 +1,20 @@
+to: 4.15.0
+from: 4[.]14[.].*
+url: https://issues.redhat.com/browse/SDN-5145
+name: OVNIPsecPausedMCPConnectivity
+message: OVN clusters with enabled IPsec with multiple worker MachineConfigPools may lose SDN connectivity during updates when at least one worker MCP is paused.
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
+        group by (ipsec) (label_replace(max_over_time(ovnkube_master_ipsec_enabled{_id=""}[1h]), "ipsec", "enabled", "", "") == 1)
+        or on (_id)
+        0 * group by (ipsec) (label_replace(max_over_time(ovnkube_master_ipsec_enabled{_id=""}[1h]), "ipsec", "disabled", "", "") == 0)
+      ) and on (_id) (
+        group by (resource) (max_over_time(apiserver_storage_objects{_id="",resource="egressips.k8s.ovn.org"}[1h]))
+        and on (_id)
+        group by (resource) (max_over_time(apiserver_storage_objects{_id="",resource="machineconfigpools.machineconfiguration.openshift.io"}[1h] ) > 2)
+      )
```

Comment:
"uses OVN" (likely redundant with "ipsec enabled" but IMO does not hurt
and
"has more than two MachineConfigPools"

It seems we cannot check pausedness so we overabstract and make false positives but with IPsec guard that may be acceptable (we do not falsely warn all clusters with custom MCPs but only ones with IPsec)